packages/gatsby-theme-aio/src/components/GetCredential/Card/AccessToken.js

```diff
@@ -20,43 +30,44 @@ const AccessToken = ({ accessToken, token }) => {
           {accessToken?.heading && (
             <h4 className="spectrum-Heading spectrum-Heading--sizeS">{accessToken?.heading}</h4>
           )}
-          {showtoken ? (
-            <div
-              css={css`
-                display: flex;
-                align-items: center;
-                gap: 16px;
-              `}>
-              <p
-                className="spectrum-Body spectrum-Body--sizeS"
-                css={css`
-                  white-space: nowrap;
-                  overflow: hidden;
-                  text-overflow: ellipsis;
-                  color: #000000;
-                  width: 320px;
-                `}>
-                {token}
-              </p>
-              <TooltipTrigger delay={0}>
-                <ActionButton onPress={() => navigator.clipboard.writeText(token)}>
-                  <CopyIcon />
-                </ActionButton>
-                <Tooltip>Copy</Tooltip>
-              </TooltipTrigger>
-            </div>
-          ) : (
-            accessToken?.buttonLabel && (
+          {showtoken ?

```

Comment:
Do we need this `showToken` variable? Can't we instead check if the `secretToken` has a value?